### ISSUE TITLE
Better syntax file?

### ISSUE THREAD

**Issue by ghost** [mitsuhiko/vim-python-combined#3]
Hi,

thanks for your plugin, I use it currently via [vim-polyglot](https://github.com/sheerun/vim-polyglot). Are you aware of [hdima/python-syntax](https://github.com/hdima/python-syntax)? It seems like a lot of the changes over there to the python syntax file would be a benefit to the users of your plugin  as well as to vim-polyglot users.

**Comment by sloria** [comment]
I agree. I have sent a PR replacing the syntax file with the one from [hdima/python-syntax](https://github.com/hdima/python-syntax).
